src/compiler/checker.ts

```diff
@@ -8697,6 +8723,37 @@ namespace ts {
             }
 
             return resolveErrorCall(node);
+            
+            function checkConstructorVisibility(signature: Signature, node: NewExpression) {
+                let constructor = result.declaration;                
+                if (!constructor || !node.expression) return;

```

Comment:
Can you wrap these `return` statements with curlies and place them on their own line?

```TypeScript
if (!constructor || !node.expression) {
    return;
}
```